schema/build.sbt

```diff
@@ -2,22 +2,22 @@ name := "codepropertygraph-schema"
 
 libraryDependencies += "io.shiftleft" %% "overflowdb-codegen" % "1.63"
 
-val generateDomainClasses = taskKey[Seq[File]]("generate overflowdb domain classes for our schema")
+scalacOptions += "-Xfatal-warnings"
```

Comment:
Is that intended?

Reply:
yup, i just moved that line within the same file